src/gt4py/backend/numpy_backend.py

```diff
@@ -313,7 +313,14 @@ def vectorized_ternary_op(*, condition, then_expr, else_expr, dtype):
 
     def generate_implementation(self):
         sources = gt_text.TextBlock(indent_size=self.TEMPLATE_INDENT_SIZE)
-        self.source_generator(self.implementation_ir, sources)
+        if self.options.backend_opts.get("ignore_np_errstate", True):
+            sources.append(
+                "with np.errstate(divide='ignore', over='ignore', under='ignore', invalid='ignore'):"
+            )
+            with sources.indented():
+                self.source_generator(self.implementation_ir, sources)
+        else:
+            self.source_generator(self.implementation_ir, sources)
```

Comment:
@egparedes Can you think of a cleaner way of doing this? I couldn't come up with anything cleaner without using a second TextBlock, which I figured would be more hassle than it's worth.

Reply:
Per our conversation: let's use `textindent.indent`.